docere/plugins/index/templates/body.html

```diff
@@ -20,7 +20,7 @@ <h1 style="margin-top:6rem">Report Index</h1>
         <a href="{{report.path}}">{{report.title}}</a>
       </td>
       <td>
-        {{report.author}}
+        {{report.author}}{{ report.authors|join(', ') }}
```

Comment:
Do we want to join a list here or just keep a string in the report.json file?

Reply:
I see two options here. One is that we keep the code as simple as possible, meaning we keep things as-is and close this PR. If you have multiple authors, they appear as a single string under key `"author"`.

If we support an `"authors"` key, I think that's naturally going to be a list of strings and we should do the join here.

Since folks may already be used to specifying a list of authors in Knowledge Repo, it seems like a nice continuity to support that in this tool. But up to you.

Reply:
SGTM! Merging as is. Thanks for humoring me